### DISCUSSION TITLE
[Projects Beta] Set fields of new issues based on the current filter

### DISCUSSION THREAD

**Discussion by mversluys** [community/community#6425] [Projects and Issues] [answered]
If you have a filter like `<field>: <value>`, when you add new issues at the moment there’s a warning displayed indicating that the new issue won’t be displayed because it doesn’t match the current filter.

Rather than reporting that warning, could all of the fields in the new issue default to the current filter values.

This will allow the new issue to show up in the existing view, and it also allows views to be used for faster issue entry.

**Comment by mrmurphy** [comment]
Yes, I find myself wanting to enter a bunch of new cards or tickets with values pre-set, like status, project, tags, and milestone, for example. A solution like this could speed up entry a lot.

**Comment by timoyates** [comment]
Thanks for the feedback folks! We're looking into this capability and hope to have it ready soon. Cheers!

**Comment by alexcnichols** [comment, marked as answer]
🚢  This has shipped! https://github.blog/changelog/2021-12-02-the-new-github-issues-12-02-update/
Give it a try and let us know what you think.

**Reply by mversluys** [reply]
Tried it out and it works great!
